code/modules/research/xenoarchaeology/artifact/effects/unknown_effect_stun.dm

```diff
@@ -35,7 +35,7 @@
 			var/weakness = GetAnomalySusceptibility(C)
 			if(prob(100 * weakness))
 				to_chat(C, "<span class='warning'>A wave of energy overwhelms your senses!</span>")
-				C.weakened += 4 * weakness
+				C.knockdown += 4 * weakness
```

Comment:
It's starting to hurt watching my laptop chug from trying to scroll down
knockdown()